commons/feature_toggles/feature_toggle_manager.py

```diff
@@ -0,0 +1,69 @@
+import os
+import json
+from time_tracker_api.security import current_user_email
+from azure.appconfiguration import AzureAppConfigurationClient
+
+
+class MSConfig:
+    def check_variables_are_defined():
+        auth_variables = ['AZURE_APP_CONFIGURATION_CONNECTION_STRING']
+        for var in auth_variables:
+            if var not in os.environ:
+                raise EnvironmentError(
+                    "{} is not defined in the environment".format(var)
+                )
+
+    check_variables_are_defined()
+    AZURE_APP_CONFIGURATION_CONNECTION_STRING = os.environ.get(
+        'AZURE_APP_CONFIGURATION_CONNECTION_STRING'
+    )
+
+
+class FeatureToggleManager:
+    def __init__(self, key: str, label: str = None, config=MSConfig):
+        self.key = key
+        self.label = label
+        self.config = config
+        self.client = self.get_azure_app_configuration_client()
+        self.configuration = {}
```

Comment:
I consider we could get rid of this field. As I mentioned in another comment https://github.com/ioet/time-tracker-backend/pull/250/files#r554218908

Reply:
It has been deleted that field.